doc/plugins/unittest.rst

```diff
@@ -0,0 +1,69 @@
+###############
+Spyder Unittest
+###############
+
+**Spyder-unittest** is a plugin that integrates popular unit test frameworks with Spyder, allowing you to run test suites and view the results in the IDE.
+
+.. image:: /images/unittest/unittest-run.png
+   :alt: Spyder Unittest in Spyder
+
+Using unittest inside Spyder allows you to take advantage of their web interface alongside Spyder’s powerful features such as the Variable explorer, console and debugger.
+
+===================
+Installing Unittest
+===================
+
+.. code-block:: bash
+
+   conda install -c spyder-ide spyder-unittest
+
+.. important::
+
+   Please make sure you have Python's pytest package installed if you are going to use that program as well.
+
+.. code-block:: bash
+
+   conda install pytest
+
+Restart Spyder in order to be able to use the plugin.
+
+==============
+Using Unittest
+==============
+
+When Unittest is installed, it will be available under the menu item :menuselection:`View --> Panes --> Unit testing`.
+
+.. image:: /images/unittest/unittest-view-panes.png
+   :alt: Spyder showing view panes Unittest
+
+You will see it then as a tab in the bottom of the top right area in the same location as the Variable explorer, Help, and Plots views.
+
+Press on the  :guilabel:`Run Green Arrow` to run the testing.
```

Comment:
```suggestion
Press on the  :guilabel:`Run Green Arrow` to run the testing.
```
I think here it might be worth to explain that we need to have some tests written in the editor and maybe mention what a test is in Python with a link to this page where there is more information about it https://docs.python.org/3/library/unittest.html. What do you think @ccordoba12 ?

Reply:
Yeah, I think that's important to explain first, i.e. before clicking `Run`, you need to have at least a file whose name starts by `test_` and then functions on it that also start by `test_`. And below that explanation, we could also show a screenshot showing that in the editor, as @juanis2112 suggested.

Of course, that's if we're going to use Pytest to show the plugin functionality, which I think is much simpler to explain than Unittest.

Reply:
The link to the Pytest website and the inclusion of the requirements for a testing file to run were included.